vscode-extension/src/extension.ts

```diff
@@ -0,0 +1,134 @@
+import {workspace, window} from 'vscode';
+
+import {
+  CloseAction,
+  ErrorAction,
+  LanguageClient,
+  LanguageClientOptions,
+  RevealOutputChannelOn,
+  ServerOptions,
+} from 'vscode-languageclient/node';
+import {findRelayBinary} from './utils';
+
+let client: LanguageClient;
+
+export async function activate() {
+  const outputChannel = window.createOutputChannel('Relay Language Server');
+
+  // TODO: Support multi folder workspaces by not using rootPath.
+  // Maybe initialize a client once for each workspace?
+  const relayBinary =
+    // TODO: Use VSCode config instead of process.env
+    process.env.RELAY_BINARY_PATH ??
+    (await findRelayBinary(workspace.rootPath));
+
+  // TODO: Use VSCode config instead of process.env
+  const outputLevel = process.env.RELAY_LSP_LOG_LEVEL ?? 'debug';
```

Comment:
same with the log-level? Should we use the value from the `configuration`?

Reply:
def in the follow up PR